nautobot_firewall_models/api/serializers.py

```diff
@@ -1,246 +1,18 @@
-"""API serializers for firewall models."""
+"""API serializers for nautobot_firewall_models."""
 
-from nautobot.apps.api import NautobotModelSerializer, ValidatedModelSerializer
-from rest_framework import serializers
+from nautobot.apps.api import NautobotModelSerializer, TaggedModelSerializerMixin
 
 from nautobot_firewall_models import models
 
 
-class IPRangeSerializer(NautobotModelSerializer):
+class IPRangeSerializer(NautobotModelSerializer, TaggedModelSerializerMixin):  # pylint: disable=too-many-ancestors
     """IPRange Serializer."""
 
-    start_address = serializers.CharField()
```

Comment:
I'm a tad concern that a large amount is being removed here. Is there a reason for removing all this?